rcjaRegistration/common/userInterface.py

```diff
@@ -0,0 +1,10 @@
+from os import environ 
+from django.shortcuts import render as django_render
+
+def render(request, template_name, context=None, *args, **kwargs):
+    """ Add development tag to header if development environment is active """
+    if type(context) == dict:
+        context["development"] = environ["DEV_SETTINGS"]
+    else:
+        context = {"development":environ["DEV_SETTINGS"]}

```

Comment:
Nice, this is a fairly effective function - as with Evan's suggestion above for staging, I can add a new environment variable to the server to accomplish this.

Let's align it with how the CMS handles environments for the sake of consistency too, there is a config called `ENVIRONMENT` which accepts "production", "staging", or "development"
If it is not set, the system falls back to "development".

I'll add this as an environment var called `ENVIRONMENT` on the production and staging instances now so that when this eventually rolls out it's already there.

You could handle the fallback inside `rcjaRegistration/rcjaRegistration/settings.py`, then like other places, import that settings file so we still manage all the environment vars centrally. 
You'll need to add the new var to the reference at the top of that file, as well as setting it in the `.env.sample`, `.env.dev.sample`, and `.env.ci` files